plugins/modules/ecs_taskdefinition.py

```diff
@@ -682,6 +686,23 @@ def register_task(self, family, task_role_arn, execution_role_arn, network_mode,
                             self.module.fail_json(msg="In awsvpc network mode, host port must be set to the same as "
                                                   "container port or not be set")
 
+            if 'linuxParameters' in container:
+                for linux_param in container.get('linuxParameters'):
+                    if linux_param == 'tmpfs':
+                        for tmpfs_param in container['linuxParameters']['tmpfs']:
+                            if 'size' in tmpfs_param:
+                                tmpfs_param['size'] = int(tmpfs_param['size'])
+
+                    for param in ('maxSwap', 'swappiness', 'sharedMemorySize'):
+                        if param in linux_param:
+                            container['linuxParameters'][param] = int(container['linuxParameters'][param])
```

Comment:
```suggestion
                    if linux_param in ('maxSwap', 'swappiness', 'sharedMemorySize'):`
                        container['linuxParameters'][param] = int(container['linuxParameters'][param])
```

Reply:
@markuman Thank you for reviewing. I used the for loop here because I need to cast to int all these parameters `('maxSwap', 'swappiness', 'sharedMemorySize')`. The user can use all together.